assets/sass/scaffolding/_grid.scss

```diff
@@ -29,6 +29,8 @@ $flexGridSpacing: $spacingUnit;
     > .flex-grid__item {
         max-width: 100%;
         margin-bottom: $flexGridSpacing;
+        list-style: none !important;
```

Comment:
Made a root cause fix to account for flex-grid being inside `s-prose` areas. Resetting the list-style and margin explicitly. Should be pretty safe but worth doing a scan of the site to confirm this doesn't cause any regressions.